routes/controller/comment.js

```diff
@@ -1,24 +1,9 @@
-const Comment = require("../../models/Comment");
-const Meal = require("../../models/Meal");
 const { OK } = require("../../constants/statusCodes");
 
-async function postComment(req, res, next) {
+function postComment(req, res, next) {
   try {
-    //mock comment
-    const newComment = await Comment.create({
-      category: "meal",
-      ratingId: "6133659530e64e9a37b9c92b",
-      creator: "6133294e0f3e1e91446fcc8c",
-      date: new Date(),
-      content: "asd",
-    });
-
-    await Meal.findByIdAndUpdate("6133659530e64e9a37b9c92b", {
-      $push: { comments: newComment._id },
-    });
-
     res.status(OK);
-    res.json({ result: "ok" });
+    res.json({ result: "ok", category: req.category });
```

Comment:
코멘트를 작성하는 시점은 디테일 페이지 접근이 완료된 시점일 것이므로, 코멘트가 등록된 후에 응답으로 카테고리를 전달해줄 필요는 없을 것 같습니다
```suggestion
    res.json({ result: "ok" });
```

Reply:
제대로 카테고리가 가는지 확인하기 위해서 쓴 코드입니다. 수정하겠습니다.